public/data/aditya10003.json

```
@@ -0,0 +1,13 @@
+{
+  "name": "Aditya kadam",
```

I recommend capitalizing the `k` here. What do you think? 🙂
```suggestion
  "name": "Aditya Kadam",
```